# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

Pr2GripperTrajActionGoal action_goal
Pr2GripperTrajActionResult action_result
Pr2GripperTrajActionFeedback action_feedback
